Upstreamed in 3e37c7d3fa122563a9d88168926f61286ef30cd3

ncurses 6.2 - patch 20200523

From 2735f43727f8cef9edb406dc361a51014807788c Mon Sep 17 00:00:00 2001
From: Anonymous Maarten <anonymous.maarten@gmail.com>
Date: Wed, 26 Feb 2020 19:24:16 +0100
Subject: [PATCH 13/15] Fix lib_gen.c

Fixes these kind of errors on MSVC
../ncurses/lib_gen.c(23): error C2059: syntax error: '('

(required for static build)

../ncurses/lib_gen.c(30): error C2059: syntax error: '('
../ncurses/lib_gen.c(37): error C2059: syntax error: '('
../ncurses/lib_gen.c(44): error C2059: syntax error: '('
../ncurses/lib_gen.c(51): error C2059: syntax error: '('
../ncurses/lib_gen.c(58): error C2059: syntax error: '('
../ncurses/lib_gen.c(65): error C2059: syntax error: '('
../ncurses/lib_gen.c(72): error C2059: syntax error: '('
../ncurses/lib_gen.c(79): error C2059: syntax error: '('
../ncurses/lib_gen.c(86): error C2059: syntax error: '('
(and more)

---
 ncurses/base/MKlib_gen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ncurses/base/MKlib_gen.sh b/ncurses/base/MKlib_gen.sh
index 5a3770e5..5e646e66 100755
--- a/ncurses/base/MKlib_gen.sh
+++ b/ncurses/base/MKlib_gen.sh
@@ -178,7 +178,7 @@ EOF3
 
 if test "$USE" = generated ; then
 cat >$ED4 <<EOF
-	s/^\(.*\) \(.*\) (\(.*\))\$/NCURSES_EXPORT(\1) (\2) (\3)/
+	s/^\(.*\) \(.*\) (\(.*\))\$/NCURSES_EXPORT(\1)\n\2 (\3)/
 	/attr_[sg]et.* z)/s,z),z GCC_UNUSED),
 EOF
 else
-- 
2.21.1

